contents/blog/posthog-changelog.md

```diff
@@ -19,6 +19,15 @@ Every Friday we update this page with the latest new features, fixes, and update
 
 > Some new features you read about below may still be in beta, behind a feature flag, or only available to paying users. Want to see which betas are currently available? [Check the roadmap](/roadmap)!
 
+## April 21, 2023
+
+#### Support triage
+![posthog support](../images/blog/array/posthog-support.png)
+
+We've made a change to the way we handle support internally in order to give you a better experience. In the app this manifests through a new 'Report bug / get support' option on the help dropdown, which opens the modal above. 
+
+Previously, we'd assign an engineer to be a [Support Hero](/handbook/engineering/support-hero) each week and they would mostly drop other work to focus on support. That worked well initially, but as PostHog has grown in breadth it's become harder for engineers to offer support outside of their usual focus. The new system automatically triages requests and assigns them to a Support Hero from the relevant small team - meaning you'll always get the best possible support. 
```

Comment:
I'd caution us here to NOT set the expectation that we'll get back to each individual, which won't be the case (details: https://posthog.slack.com/archives/C0351B1DMUY/p1681735739348339) 
I'd instead focus on the fact that we can start using aggregated info (we can say that the amount of requests has become too much for our small team to handle & still ship new exciting things), e.g. funnels getting a lot of bug reports - let's take a week to work most common issues.